macros/InterfaceData.json

```diff
@@ -1723,6 +1743,10 @@
     "inh"  : "Event",
     "impl" : []
   },
+  "RTCPeerConnectionIceErrorEvent": {
```

Comment:
This is not in Blink or Gecko idls. AFAICT it is implemented nowhere and so we don't document it yet, and it shouldn’t be here. (specs change, we document reality)